Packs/FeedCyrenThreatInDepth/Integrations/CyrenThreatInDepth/CyrenThreatInDepth.py

```diff
@@ -0,0 +1,396 @@
+}
+
+
+class Client(BaseClient):
+    PARAMS = {"format": "jsonl"}
+
+    def __init__(self, feed_name: str, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.feed_name = feed_name
+
+    def _do_request(self, path: str, offset: int = -1, count: int = 0) -> requests.Response:
+        params = self.PARAMS.copy()
+        params["feedId"] = self.feed_name
+        if offset > -1:
+            params["offset"] = str(offset)
+        if count > 0:
+            params["count"] = str(count)
+
+        demisto.info(f"using path {path}, params {params} for request")
+
+        try:
+            response = self._http_request(method="GET", url_suffix=path,
+                                          params=params, resp_type="")
+        except requests.ConnectionError as e:
+            raise requests.ConnectionError(f"Failed to establish a new connection: {str(e)}")
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError:
+            demisto.error(f"exception in request: {response.status_code} {response.content!r} {path}")
+            raise
+
+        return response
+
+    def fetch_entries(self, offset: int, count: int) -> List[Dict]:
+        result = []
+        response = self._do_request(path=FeedPath.DATA, offset=offset, count=count)
+
+        for json_line in response.content.splitlines():
+            try:
+                result.append(json.loads(json_line))
+            except json.JSONDecodeError as je:
+                demisto.info(f"error while parsing single JSON feed line: {str(je)} {json_line!r}")
+                continue
+
+        return result
+
+    def get_offsets(self) -> Dict[str, int]:
+        response = self._do_request(path=FeedPath.INFO)
+        try:
+            return json.loads(response.content)
+        except Exception as e:
+            demisto.error(f"error while parsing JSON: {str(e)} {response.content!r}")
+            raise
+
+
+def test_module_command(client: Client) -> str:
+    try:
+        entries = client.fetch_entries(0, 10)
+        if not any(entries):
+            return "Test failed because no indicators could be fetched!"
+    except Exception as e:
+        return f"Test failed because of: {str(e)}!"
+
+    return "ok"
+
+
+def get_indicators_command(client: Client, args: Dict) -> Tuple[str, Dict, List]:
+    max_indicators = int(args.get("max_indicators", 50))
+
+    count = max_indicators
+    end_offset = client.get_offsets().get("endOffset")
+    if end_offset:
+        offset = end_offset - max_indicators + 1
+    else:
+        offset = 0
+
+    entries = client.fetch_entries(offset, count)
+    indicators, _ = feed_entries_to_indicator(entries, client.feed_name)
+
+    human_readable = tableToMarkdown("Indicators from Cyren Threat InDepth:", indicators,
+                                     headers=["value", "type", "rawJSON", "score"])
+    return human_readable, dict(), indicators
```

Comment:
I tried! Can you be more specific? Especially around which of the three goes into which field of `CommandResults`...

Reply:
Sure, thing:
Given the above code:
```
CommandResults(
    readable_output=human_readable,
    raw_response=indicators
)
```

Since the outputs you provide is empty, you shouldn't bother passing it anyway.